src/stk/databases/mongo_db/constructed_molecule.py

```diff
@@ -48,6 +48,18 @@ class ConstructedMoleculeMongoDb(ConstructedMoleculeDatabase):
             )
         )
 
+        # Change the database MongoClient will connect to.
```

Comment:
Is this snippet in all examples now? Just seeing it pop up a lot?

Reply:
In all the examples that use `MongoClient`